src/components/organisms/Table/v2/Badge.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// T Y P E S
+
+export interface BadgeProps {
+  value: string | boolean | number
+}
+
+// C O M P O N E N T S
+
+export const Badge: React.FC<BadgeProps> = (props) => {
+  const { value } = props
+  return (
+    <StyledBadge>{value.toString()}</StyledBadge>
```

Comment:
```suggestion
export const Badge: React.FC<BadgeProps> = (props) => {
  const { value, className } = props
  return (
    <StyledBadge className={className}>{value}</StyledBadge>
```

Please don't forget to handle styled-components by having a `className` prop. Also I guess you don't need to use here the `.toString()` method 👍

Reply:
i'll add the 'className' props, but 'toString()' is required for boolean values.

Reply:
But what do you expect as a user when you give the props `value={true}`? That the badge contains `true`? `1`? To avoid this kind of confusion don't let the user the choice. Just handle `value: string | JSX.Element`.